js/client-teambuilder.js

```diff
@@ -1047,7 +1047,11 @@
 			};
 			buf += '<span class="detailcell detailcell-first"><label>Level</label>' + (set.level || 100) + '</span>';
 			if (this.curTeam.gen > 1) {
-				buf += '<span class="detailcell"><label>Gender</label>' + GenderChart[template.gender || set.gender || 'N'] + '</span>';
+				if (this.curTeam.format !== 'balancedhackmons') {
+					buf += '<span class="detailcell"><label>Gender</label>' + GenderChart[template.gender || set.gender || 'N'] + '</span>';
+				} else {
+					buf += '<span class="detailcell"><label>Gender</label>' + GenderChart[set.gender || 'N' || template.gender] + '</span>';
```

Comment:
Should `'N'` go before `template.gender`? I think it should be the other way around...